app/javascript/controllers/index.js

```diff
@@ -1,11 +1,23 @@
-// Import and register all your controllers from the importmap under controllers/*
+// This file is auto-generated by ./bin/rails stimulus:manifest:update
+// Run that command whenever you add a new controller or create them with
+// ./bin/rails generate stimulus controllerName
 
-import { application } from "controllers/application"
+import { application } from "./application"
```

Comment:
This diff feels like a big step back. Is there anything we can do to restore the autoloading to work correctly?

Reply:
I think it's a tradeoff. This file is auto generated using `rake stimulus:manifest:update`. When it's generated like this, the files seem to correctly refresh when changes are made to the controllers. Without this change, you usually have to restart the rails server to get sprockets to pickup the change. The trade-off is that when you need to add a new controller or remove one, you'll need to run that rake command.

Propshaft works better, but I had problems getting it to build reliably. Staging was broken last time I tried to deploy it.

Reply:
I removed this change from this PR. I think it's worth considering adding it separately or trying to move to propshaft again.